etc/conda-install.sh

```diff
@@ -59,8 +58,8 @@ RECREATE_CONDA_ENV=0
 
 create-conda-env() {
     # create a new environment
-    mamba update -n base -c defaults conda
     mamba update -n base -c conda-forge mamba
+    mamba update -n base -c conda-forge conda
```

Comment:
IMO we should be good updating mamba and conda in the same command unless I'm missing something; also we can remove the `-c` option unless we want to be explicit:

```suggestion
    mamba update -n base mamba conda
```

Reply:
I actually ran into a failure when both were updated in the same command. It pulled a new conda and installed that before installing the new mamba, which broke it. That’s why I put the mamba update first. I could have missed something else, but I believe that’s what occurred.

Reply:
Ah okay in that case this method is fine by me!

Reply:
I can test this again, or we can merge it as-is and try that again later. The safer bet is two separate commands until we try to reproduce the problem I saw.